.github/workflows/test.yml

```diff
@@ -172,9 +172,40 @@ jobs:
             build-type: "Debug"
           - name: "Push Release Docker image"
             build-type: "Release"
+    runs-on: ubuntu-latest
+    container: soramitsu/kagome-dev:4-minideb
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
please, pin version here and in all other actions

Reply:
Done